content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx

```diff
@@ -0,0 +1,321 @@
+---
+title: 'Upgrade to Prisma 5'
+metaTitle: 'Upgrade to Prisma 5'
+metaDescription: 'Guides on how to upgrade to Prisma 5'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+Prisma 5.0.0 introduces a number of **breaking changes**. This guide explains how upgrading might affect your application and gives instructions on how to handle any breaking changes.
+
+</TopBlock>
+
+## Major changes
```

Comment:
Major/minor might be difficult terminology choice, as Prisma 5 itself is a "major" version increase vs, the usual "minor" releases we ship.

Reply:
Is the terminology "Major breaking changes" and "Minor breaking changes" too verbose? Originally I had this section as "breaking changes" but our Notion doc has the section breaking (major and minor) and non-breaking

Reply:
Yeah the Notion was trying to categorize by impact, and chose not great terminology for that.

I think "major breaking change" and "minor breaking changes" are indeed bad as we are shipping a "major version".